lib/danger/scm_source/git_repo.rb

```diff
@@ -11,8 +11,11 @@ def diff_for_folder(folder, from: "master", to: "HEAD", lookup_top_level: false)
       git_top_level = folder
       if lookup_top_level
         Dir.chdir(folder) do
-          git_top_level = exec("rev-parse --show-toplevel")
+          git_top_level = Git::Base.root_of_worktree(folder)
         end
+      elsif git_top_level != Git::Base.root_of_worktree(folder)
```

Comment:
`Git::Base.root_of_worktree` is unavailable at `git` gem v1.13.0. I will rework this.